trieste/acquisition/rule.py

```diff
@@ -79,33 +79,36 @@ def acquire(
             :class:`~trieste.space.SearchSpace` depends on the specific :class:`AcquisitionRule`.
 
         :param search_space: The local acquisition search space for *this step*.
-        :param datasets: The known observer query points and observations for each tag.
-        :param models: The model to use for each :class:`~trieste.data.Dataset` in ``datasets``
-            (matched by tag).
+        :param models: The model for each tag.
+        :param datasets: The known observer query points and observations for each tag. (optional)
```

Comment:
Is it worth saying that the necessity of passing datasets is specified by the chosen acquisition function?

Reply:
I agree, it would be useful to clarify what optional means

Reply:
Strictly speaking it's up to the rule, as the rule may not even use an acquisition function! But I can add that to the EGO docstring, and include a more generic explanation here (similar to the comment about functions above).